### PR TITLE
Reduce time to join Slack

### PR DESCRIPTION
**Pull request by sampoder**
This is slightly opinionated, @maxwofford I'd like your approval before merging. Merging this PR will deprecate the Join Requests Table in Airtable and will also not await the post request to Toriel (as we weren't doing anything with that response data anyway)

### CODE REVIEW

**Comment by sampoder** [comment]
closes #414

File: pages/api/join.js [hunk 1]
```diff
@@ -57,18 +57,8 @@ export default async function handler(req, res) {
...
-    Student: data.educationLevel != 'tertiary' ? true : false,
-    Reason: data.reason,
-    Invited: open,
-    Club: data.club ? data.club : '',
-    IP: req.headers['x-forwarded-for'] || req.socket.remoteAddress
-  })
-
   if (open) {
-    let result = await postData(
+    postData(
```
Comment (maxwofford) [inline pages/api/join.js]: Unfortunately we have to await this– if we fire-and-forget and res.json, we might close this connection: https://docs-git-add-fire-and-forget-zeit.vercel.app/docs/v2/platform/limits?query=fire-and-forget#streaming-responses

File: pages/api/join.js [hunk 2]
```diff
@@ -57,18 +57,8 @@ export default async function handler(req, res) {
     }
   }
 
-  await joinTable.create({
```
Comment (maxwofford) [inline pages/api/join.js:60]: Let's keep this around for now– at the very least it lets us keep a waitlist & prevents us from breaking another system under the hood that relies on this but forgot. To speed this up, what if we run this in parallel (`Promise.all`) with the toriel endpoint?

File: pages/api/join.js (hunk 1 above)
Comment (sampoder) [reply pages/api/join.js]: ugh no fun!!

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:



- [**/api/join**](https://site-6iah6sp1r.hackclub.dev/api/join)



Commit 2321b642214c3d5e94625f48298a9fb8fe464b07 (https://site-6iah6sp1r.hackclub.dev).

**Pull request merged**